src/d3a/models/strategy/area_agents/two_sided_pay_as_clear_engine.py

```diff
@@ -161,16 +161,22 @@ def _match_offers_bids(self):
             self._delete_forwarded_offer_entries(offer)
 
     def tick(self, *, area):
-        super().tick(area=area)
 
         for bid_id, bid in self.markets.source.bids.items():
             if bid_id not in self.forwarded_bids and \
                     self.owner.usable_bid(bid) and \
                     self.owner.name != bid.seller:
                 self._forward_bid(bid)
+
```

Comment:
This copy-pastes code, and apart from that it is still not succeeding, since there is a whole bunch of code that is related to offer forwarding that is not included here. Same applies to the 5 lines above this change, it is just for the two sided pay as bid market, and should not be repeated. Like in the previous comment, I would either refactor the code with function overloading to do exactly what each layer is supposed to do (offer forwarding in the one sided engine, bid forwarding in the pay as bid engine), OR to call the one sided engine constructor directly from here.